tests/unit/bases/test_ubuntu_buildd.py

```diff
@@ -685,7 +685,7 @@ def test_pre_setup_packages_devel(fake_executor, fake_process, mocker):
 
 
 def test_ensure_image_version_compatible_failure(fake_executor, monkeypatch):
-    base_config = ubuntu.BuilddBase(alias=ubuntu.BuilddBaseAlias.FOCAL)
+    base_config = ubuntu.BuilddBase(alias=ubuntu.BuilddBaseAlias.JAMMY)
```

Comment:
It might be worth making a general fixture for this (e.g. `buildd_stable_base`) so future updates don't have to change so many lines. (Probably best for a separate PR. I'm happy to do so but I'd like your opinion first)